cmake/modules/hunter_create_cache_file.cmake

```diff
@@ -51,6 +52,58 @@ function(hunter_create_cache_file cache_path)
       "set(HUNTER_STATUS_DEBUG \"${HUNTER_STATUS_DEBUG}\" CACHE INTERNAL \"\")\n"
   )
 
+  # Pass compiler ABI info through to avoid recalculating in every project
+  foreach(lang C CXX)
+    if(DEFINED CMAKE_${lang}_SIZEOF_DATA_PTR)
```

Comment:
Why we need to check that variable is defined? May be just set it to cache unconditionally?